### ISSUE TITLE
Error importing curry on django==3.0

### ISSUE THREAD

**Issue by nl-emmanuellodovice** [saxix/django-adminactions#163]
`ImportError: cannot import name 'curry' from 'django.utils.functional' (lib/python3.7/site-packages/django/utils/functional.py)`

**Comment by mattpavelle** [comment]
Yeah - curry has been removed from Django 3.0 (see: https://docs.djangoproject.com/en/3.0/_modules/django/utils/functional/)

I'm not a contributor - just pointing out that this is due to changes in Django, not a bug in this project.

**Comment by marcasteroid** [comment]
I'm trying to use 

> django-rest-framework-social-oauth2==1.1.0

but the the error

```
from django.utils.functional import curry, Promise
ImportError: cannot import name 'curry' from 'django.utils.functional'
```

What to do now ?

**Comment by mattpavelle** [comment]
@marcasteroid wait until the author has time to rewrite the pieces that require Curry, or use Django 2

**Comment by emanlodovice** [comment]
seems to be fixed in https://github.com/saxix/django-adminactions/commit/18451a0ba07485844faf4cebba9f4a09fa3a05a2#diff-ad79f90c12c370ca9724f78e0e072d96

**Comment by saxix** [comment]
fixed in 1.8.1
